ui/src/app/subnets/views/SubnetsList.tsx

```diff
@@ -1,13 +1,47 @@
 import React from "react";
 
-const SubnetsList = (): React.ReactElement => (
-  <header className="p-strip--light is-shallow u-no-padding--bottom">
-    <div className="row">
-      <div className="col-medium-4 col-8">
-        <h1 data-testid="section-header-title">Subnets</h1>
-      </div>
-    </div>
-  </header>
-);
+import { ContextualMenu } from "@canonical/react-components";
+
+import Section from "app/base/components/Section";
+import SectionHeader from "app/base/components/SectionHeader";
+import { SubnetForms } from "app/subnets/enum";
+import type { SubnetForm } from "app/subnets/types";
+import FormActions from "app/subnets/views/FormActions";
+
+const SubnetsList = (): React.ReactElement => {
+  const [activeForm, setActiveForm] = React.useState<SubnetForm | undefined>(
+    undefined
+  );
```

Comment:
We conventionally use `null` when we want to define a variable as having "no value", i.e. we consider `null` to mean the reference exists but is (currently) empty, and `undefined` to mean the reference does not exist.